applications/main/bad_usb/bad_usb_script.c

```diff
@@ -204,16 +207,29 @@ static bool ducky_altstring(const char* param) {
     return state;
 }
 
-static bool ducky_string(const char* param) {
+static bool ducky_string(BadUsbScript* bad_usb, const char* param) {
     uint32_t i = 0;
+    uint32_t timing = 0;
+    
+    if(bad_usb->stringdelay == 0) {
+        timing = 0;
+    } else if(bad_usb->stringdelay > 1000) {
+        timing = bad_usb->stringdelay / 2 / 10;
```

Comment:
Not very obvious, it will great to describe it in badusb script documentation (documentation/file_format/BadUsbScriptFormat.md)

Reply:
added entry, thank you for pointing out